tasks/setup_requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Install libselinux-python package for 'copy' task
+  yum:
+    name: libselinux-python
+    state: present
+  when: ansible_os_family == "RedHat"
```

Comment:
It should be handled for other systems as well ... but I think that name of package differs ...